controls/radgridview/features/gridview-new-row.md

```diff
@@ -0,0 +1,153 @@
+---
+title: New Row
```

Comment:
The name is a bit vague. I would use something like "Add New Row Button" or "New Row Button", or "Add Row Button".